bin/wp-enforcer

```diff
@@ -29,30 +38,33 @@ do
       echo "  wp-enforcer [options]"
       echo ""
       echo "OPTIONS:"
-      echo "  --help, -h          Show this help and usage"
-      echo "  --version, -v       Show the current version of WP Enforcer"
-      echo "  --vip               Configure the project to use the WordPress.com VIP coding standards"
-      echo "  --customruleset, -c Configure the project to use a custom PHPCS ruleset file"
+      echo "  --help, -h               Show this help and usage"
+      echo "  --version, -v            Show the current version of WP Enforcer"
+      echo "  --vip                    Configure the project to use the WordPress.com VIP coding standards"
+      echo "  --customruleset=FILE, -c Configure the project to use a custom PHPCS ruleset file"
```

Comment:
Thoughts on just naming this --ruleset?

Also thinking we should stick with long option names for now, as we are still building out the feature set. Would hate to adopt a character for a specific option when we haven't fully identified potential features.  This would also match the usage for --vip with no additional command, since -v is already taken.